GraphModelDraw/EdgeDraw.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GraphModelDraw
+{
+    public class EdgeDraw
+    {
+        public readonly BrushColor BrushEdge;
+
+        public readonly int WeightEdge;
+
+        public readonly int Id;
+
+        public readonly int ConnectabelVertex;
+
```

Comment:
There is no need to describe the weight in the model drawing.It is necessary to create a field for the Id of the vertex, from which it comes and to which it is directed